bodhi-client/bodhi-client.spec

```diff
@@ -0,0 +1,44 @@
+# Created by pyp2rpm-3.3.7
+%global pypi_name bodhi-client
+%global pypi_version 5.7.4+devel
+
+Name:           %{pypi_name}
+Version:        %{pypi_version}
+Release:        1%{?dist}
+Summary:        Bodhi client
+
+License:        GPLv2+
+URL:            https://github.com/fedora-infra/bodhi
+Source0:        %{pypi_name}-%{pypi_version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(click)
+BuildRequires:  python3dist(koji)
+BuildRequires:  python3dist(python-fedora) >= 0.9
+BuildRequires:  python3dist(setuptools)
+
+%description
+
+
+%prep
+%autosetup -n %{pypi_name}-%{pypi_version}
+# Remove bundled egg-info
+rm -rf %{pypi_name}.egg-info
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n %{pypi_name}
+%exclude %{python3_sitelib}/tests
```

Comment:
That should not be necessary, it means the `setup.cfg` file should be fixed to exclude the tests directory, I'll fix it.

Reply:
Fixed by #4372, you can rebase and remove this line now.

Reply:
done